docs/implement/user-attributes/README.md

```diff
@@ -82,3 +82,31 @@ of the more common ways is to declare them in a Spring XML configuration file wi
 
 [independent project with its own source code called PersonDirectory]: https://github.com/apereo/person-directory
 [uPortal-start]: https://github.com/Jasig/uPortal-start
+
+## Personalization
+
+As of uPortal `5.10.0`, uPortal supports token replacement in various portlet definition fields of user attributes (known as 'personalization'). Personalization consists of a filter for all `/api` JSON GET requests, and targeted areas of the UX that leverage endpoints outside of the `/api` context. The exception to the `/api` coverage is `/layoutDoc`, which is deprecated.
+
+By default, the filter is enabled.  To disable, add the following into `uPortal.properties`:
+```
```

Comment:
```suggestion
```properties
```

Reply:
@cbeach47 have you pushed the commit that include this change?
I'm not seeing it on GH yet :thinking:

Reply:
ah - realized this morning that I didn't pick up the README changes in that push.  I've pushed the README changes now.